databuilder/backends/tpp.py

```diff
@@ -224,3 +224,28 @@ class TPPBackend(BaseBackend):
             last_linkage_dt="last_linkage_dt",
         ),
     )
+
+    isaric = QueryTable(
```

Comment:
Given that you're not doing any transformations on this table, just renaming columns, you might be better off with a `MappedTable`. There's an example directly above of what that looks like.

Reply:
I was thinking of that, then ended up writing a query to cast various types and then just stuck with that.

No problem to switch to this.

Reply:
Ah yeah, it's annoying that we have to drop down to raw SQL just to cast columns. I think we could actually add support for casting (and other per-column transformations generally) to `MappedTable`, I just haven't got round to it.